cloudapi/logs.go

```diff
@@ -166,6 +173,18 @@ func (c *Config) StreamLogsToLogger(
 		if err != nil {
 			logger.WithError(err).Warn("error reading a message from the cloud")
 
+			// try to restore the stream establishing a new connection
+			logger.Warn("trying to establish a fresh connection with the tail logs, this might result in either some repeated or missed messages") //nolint:lll
```

Comment:
I think we could remove or rephrase it, because with the current solution `some repeated or missed messages` shouldn't really happen.

Reply:
why not combine both warnings? something like this
```go
logger.WithError(err).Warn("error reading a log message from the cloud, trying to establish a fresh connection with the logs service...")
```